pkg/internal/infrastructure/terraform.go

```diff
@@ -135,6 +139,29 @@ func ComputeTerraformerChartValues(
 		},
 	}
 
+	if config.Networks.CloudNAT != nil && config.Networks.CloudNAT.NatIPNames != nil {
```

Comment:
Why don't we move these computations into the if condition after line 86 where we already check for non-nil `CloudNAT`? Then we could also easier compute the `networks` value and probably don't need the type casting in line 162.

Reply:
That's true, but we only want to append those values to the values map if `config.Networks.CloudNAT.NatIPNames != nil` or i'm wrong? Then we have to append it after calculating the values.

Reply:
Calculated it now before and appended it afterwards.